ls1intum/Artemis
# [Exam] Navigation button status

### Checklist
- [ ] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server

### Motivation and Context


When starting start and exam, all buttons in the navigation bar have the “saved icon”, even if you have not yet worked on them.
The checkmark icon should only appear when the student has worked on it and it was at least saved once.

### Description

Submissions are now initialized as not submitted and and the 'checkmark icon' is only shown when submitted.

### Steps for Testing


1. Log in to Artemis
2. Prepare an exam with some exercises
3. Participate in the exam
4. Check that the 'edit icon' is displayed on start and only if the exercise is edited
5. Check that the navigation buttons still indicate the correct status

### Screenshots


![image](https://user-images.githubusercontent.com/51987021/87290178-f4bcb800-c4fd-11ea-98f9-9e51caa87698.png)
